x-pack/filebeat/module/cisco/asa/test/asa-fix.log-expected.json

```diff
@@ -19,6 +19,7 @@
         "event.kind": "event",
         "event.module": "cisco",
         "event.original": "%ASA-6-302016: Teardown UDP connection 110577675 for Outside:10.123.123.123/53723(LOCAL\\Elastic) to Inside:10.233.123.123/53 duration 0:00:00 bytes 148 (zzzzzz)",
+        "event.reason": "(zzzzzz)",
```

Comment:
Is this just because of testdata or expected to be a reason?

Reply:
Just commited a fix for this - one of the patterns in the list was out of order.